### DISCUSSION TITLE
`Blaze Away` is overpowered

### DISCUSSION THREAD

**Discussion by scarf005** [scarf005/Marisa#88] [Balance]
~~blaze away OP, nerf plz~~
## Why OP
1. since replayed attack from `Blaze Away` acts same as playing from hand:
- playing `Blaze Away` gives 2 charges (itself + replayed attack)
- triggers `Amplification Wand` when it replays amplified attack
2. when upgraded, it replays attack **two** times.

overall, it's pretty powerful for 1 energy, and crazy powerful when upgraded

### make replayed attack don't charge && don't apply wand power
related code is still spaghetti and won't be easy to debug

### increase cost to 2
will make card unusable

### leave it as be
I like my card overpowered

**Comment by erasels** [comment]
What about making it exhaust? Would make it more of a payoff card to replay a Final Spark or something like that.

**Reply by scarf005** [reply]
that looks like a good suggestion.

**Reply by scarf005** [reply]
fixed in #171.
